Adafruit_SSD1306.h

```diff
@@ -130,6 +130,8 @@ class Adafruit_SSD1306 : public Adafruit_GFX {
 
   ~Adafruit_SSD1306(void);
 
+  enum SSD1306_Colors { BLACK=0, WHITE=1, INVERSE=2 };
```

Comment:
If I'm following along correctly, the reason for this change was that another module used a `#define BLACK 0`. Well if that situation arises again, such a macro definition would still affect this header the same way and the problem remains: you're still using the token `BLACK`.

If you're going to bring these into C++ from the preprocessor, which is a good idea, maybe give them names that won't conflict with the macro all-caps convention?

Maybe change to Black, White, Inverse.

Also if your enum is nested inside a class called Adafruit_SSD1306, you shouldn't have to say SSD1306 again in the name: `enum Adafruit_SSD1306::SSD1306_Colors`. Just `enum Adafruit_SSD1306::Colors` seems better.